src/style.css

```diff
@@ -1 +1,120 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  position: relative;
+  display: flex;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  align-items: center;
+  width: calc(100% - 2px);
+  height: 68px;
+  margin-top: 20px;
+}
+
+.search:hover {
+  box-shadow: 0 1px 8px #3d4e611a;
+}
+
+.search_icon {
+  background: url(./images/Search.svg) no-repeat;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  position: absolute;
+}
+
+.search_form {
+  width: calc(100% - 62px);
+  height: 22px;
+  margin-left: 62px;
+  padding: 0;
+  position: absolute;
+}
+
+.search_input {
+  color: #3d4e61;
+  border-color: #0000;
+  border: none;
+  width: calc(100% - 62px);
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.search_input:focus {
+  font-weight: bold;
+}
+
+.search-mini {
+  display: flex;
```

Comment:
Consider using a basic class with common styles and an additional class to add styles that are different